src/Wallabag/CoreBundle/Resources/views/base.html.twig

```diff
@@ -70,10 +70,6 @@
 
         {% block footer %}{% endblock %}
 
-        <div id="warning_message">
```

Comment:
> (In the future, at least)

We'll add the `<div>`. I'm not fan of the "and if ever...". If we'll need it, we'll add it.